install/conf/srm-toolchain/alertmanager/config.yml

```diff
@@ -0,0 +1,6 @@
+route:
+ receiver: 'delfin'
```

Comment:
Why dont we keep the project specific configuration files with separate folders? Same for related files below as well.

like ....dashboard/install/conf/delfin/.......
.......conf/strato/....

Reply:
This is done. 
currently the structure is changed to 
```
conf/
     - keystone/
         ...
     - delfin
         ... 
```